### DISCUSSION TITLE
How to give type to button in Mantine V7

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mantinedev/mantine#5801] [Q&A] [answered]
```
import { Button } from '@mantine/core';
<form onSubmit={() => handleSubmit()}>
  < Button type="submit" >
      Apply Filter
  </Button>
</form>
```

Html form element requires button to have a type, but I am using Mantinve V7.5.2 and try to give type it gives me the following error

`Type '{ children: string; type: string; disabled: boolean; color: string; }' is not assignable to type 'IntrinsicAttributes & ButtonProps'.
  Property 'type' does not exist on type 'IntrinsicAttributes & ButtonProps'.`
  
  Any help is appreciated, Thanks in advance

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/mantine-react-template-forked-4kvvq7?file=/src/App.tsx

**Reply by lokeshwarobuli** [reply]
Thanks a lot @rtivital !!
